Description: Fixes DEST_MODULE_LOCATION, otherwise there's unowned files after purge (policy 6.8, 10.8)
Author: Andreas Beckmann <debian@abeckmann.de>
Forwarded: no

--- blktap-dkms-2.0.91.orig/dkms.conf
+++ blktap-dkms-2.0.91/dkms.conf
@@ -4,4 +4,4 @@ CLEAN="make -C ${kernel_source_dir} M=${
 MAKE="make -C ${kernel_source_dir} M=${dkms_tree}/${PACKAGE_NAME}/${PACKAGE_VERSION}/build modules"
 AUTOINSTALL="yes"
 BUILT_MODULE_NAME="blktap"
-DEST_MODULE_LOCATION="/kernel/../extra/"
+DEST_MODULE_LOCATION="/extra"
